limseung1/coursescheduler
# User manual dialog

Add user manual dialog

WHAT/  HOW
- Refactored Drag and Drop Dialog to use base component named DialogBase
- Created Dialog for tutorial
- Added GIFs that explains how to use the app

TEST
- Open the app > Observe Dialog appears
- Open the app > Close Dialog > Refresh > Observe Dialog appears
- Open the app > Click Do not show again > Close Dialog > Refresh > Do not observe Dialog appear
